### PR TITLE
[DC-1088] Bash script sandbox update

### PR DESCRIPTION
**Pull request by nishanthpp93**
* Updated bash scripts to create sandbox with the appropriate labels

### CODE REVIEW

**Review by ChaoPang — COMMENTED** [review]
A general comment, it seems that we create `dataset_staging_sandbox` as well as `dataset_sandbox`, we first create all the sandboxed tables in `dataset_staging_sandbox` when running `clean_cdr.py`, then copy the content from `dataset_staging_sandbox` to `dataset_sandbox`. I think that's because previously `sandbox_dataset` used to be coupled with the corresponding `dataset` because it's automatically inferred by `check_and_create_sandbox_dataset`. Now that we have decoupled the two datasets, I don't think we need to create `dataset_staging_sandbox` anymore. When calling `clean_cdr.py`, we could just use `dataset_sandbox` directly.

**Comment by nishanthpp93** [comment]
Thanks for looking at it @ChaoPang. This would be a significant change from our current process. I am just following precedence that was already agreed upon in https://github.com/all-of-us/curation/pull/615/commits/5031eb1e75b0c021ed5b953ad7bcc745ee1511fb that was reviewed and merged recently after discussion on the eng sync channel. If we need to change this, we need to start a new discussion.

File: data_steward/tools/deid_runner.sh [hunk 1]
```diff
@@ -133,7 +144,10 @@ bq update --description "${version} De-identified Base version of ${cdr_id}" --s
...
 
 #copy sandbox dataset
-"${TOOLS_DIR}/table_copy.sh" --source_app_id ${APP_ID} --target_app_id ${APP_ID} --source_dataset "${cdr_deid_base_staging}_sandbox" --target_dataset "${cdr_deid_base}_sandbox"
+"${TOOLS_DIR}/table_copy.sh" --source_app_id ${APP_ID} --target_app_id ${APP_ID} --source_dataset "${cdr_deid_base_staging_sandbox}" --target_dataset "${cdr_deid_base_sandbox}"
+
+# Update sandbox description
+bq update --description "Sandbox created for storing records affected by the cleaning rules applied to ${cdr_deid_base}" --set_label "phase:sandbox" --set_label "release_tag:${dataset_release_tag}" --set_label "de_identified:true" "${APP_ID}":"${cdr_deid_base_sandbox}"
 
 # remove intermediary datasets
 bq rm -r -d "${cdr_deid_base_staging}_sandbox"
```
Comment (ChaoPang) [inline data_steward/tools/deid_runner.sh:153]: Use the environment `${cdr_deid_base_staging_sandbox}`

File: data_steward/tools/deid_runner.sh [hunk 2]
```diff
@@ -142,23 +156,29 @@ bq rm -r -d "${cdr_deid_base_staging}"
...
 
 #copy sandbox dataset
-"${TOOLS_DIR}/table_copy.sh" --source_app_id ${APP_ID} --target_app_id ${APP_ID} --source_dataset "${cdr_deid_clean_staging}_sandbox" --target_dataset "${cdr_deid_clean}_sandbox"
+"${TOOLS_DIR}/table_copy.sh" --source_app_id ${APP_ID} --target_app_id ${APP_ID} --source_dataset "${cdr_deid_clean_staging_sandbox}" --target_dataset "${cdr_deid_clean_sandbox}"
+
+# Update sandbox description
+bq update --description "Sandbox created for storing records affected by the cleaning rules applied to ${cdr_deid_clean}" --set_label "phase:sandbox" --set_label "release_tag:${dataset_release_tag}" --set_label "de_identified:true" "${APP_ID}":"${cdr_deid_clean_sandbox}"
 
 # remove intermediary datasets
 bq rm -r -d "${cdr_deid_clean_staging}_sandbox"
```
Comment (ChaoPang) [inline data_steward/tools/deid_runner.sh:184]: Use the environment variable `${cdr_deid_clean_staging_sandbox}`

**Review by ChaoPang — CHANGES_REQUESTED** [review]
Everything looks good except two minor comments.

**Review by ChaoPang — APPROVED** [review]
Looks good now

**Pull request merged**